mesonbuild/build.py

```diff
@@ -626,10 +626,13 @@ def add_external_deps(self, deps):
                 # about the interpreter so we can't import it and use isinstance.
                 # This should be reliable enough.
                 if hasattr(dep, 'subproject'):
-                    raise InvalidArguments('''Tried to use subproject object as a dependency.
-You probably wanted to use a dependency declared in it instead. Access it
-by calling get_variable() on the subproject object.''')
-                raise InvalidArguments('Argument is not an external dependency.')
+                    raise InvalidArguments('Tried to use subproject object as a dependency. ' \
```

Comment:
This is not equivalent since the former included the newlines and this one doesn't. Also, the backslash at the end is not necessary here.